howto/port/install-as-system-app.md

```diff
@@ -0,0 +1,75 @@
+Normally, a user app is signed by the developer and restricted to fewer permissions at runtime while a system app is usually [signed with the platform key](https://source.android.com/devices/tech/ota/sign_builds) when building an Android image, it's pre-installed under the system partition and running with some ["signature" protection level permissions](https://developer.android.com/guide/topics/manifest/permission-element.html#plevel) in the system process of Android container. An application must be running as a system app if it
```

Comment:
The `in the system process` is not correct as that indicates that the app runs as part of the `system_server` process which it does not. It's more running as independent process but with system privileges.

Reply:
+1